test/models/ping_thing_recent_stat_test.rb

```diff
@@ -0,0 +1,82 @@
+# frozen_string_literal: true
+
+require "test_helper"
+
+class PingThingRecentStatTest < ActiveSupport::TestCase
+  include ActionCable::TestHelper
```

Comment:
Is it needed?

Reply:
Will be in a follow-up task, so I added it to remember to add a test as well.